src/Components/Buttons/PrimaryButton.js

```diff
@@ -2,7 +2,7 @@ import React from "react";
 
 const PrimaryButton = (props) => {
   return (
-    <button {...props} className={props.className + " btn"}>
+    <button {...props} className={props.className + " "} >
```

Comment:
don't need the empty string, this could just be
```suggestion
    <button {...props} className={props.className}>
```